### PR TITLE
[HOLD] [v2] Cleanup AggregationCount field naming.

### PR DESCRIPTION
**Pull request by alloy**
Part of: https://artsyproduct.atlassian.net/browse/PLATFORM-1683

As [discussed in Slack](https://artsy.slack.com/archives/C1HH3KNJG/p1566396382002700).

### CODE REVIEW

**Comment by alloy** [comment]
Perhaps don't merge it yet, I'll do so myself when I have an accompanying Emission PR up.

**Comment by mzikherman** [comment]
Nice! LGTM.

File: src/schema/v2/aggregations/aggregation_count.ts [hunk 1]
```diff
@@ -12,17 +12,20 @@ export const AggregationCountType = new GraphQLObjectType<any, ResolverContext>(
...
       },
       name: {
-        type: GraphQLString,
-      },
-      sortableID: {
-        type: GraphQLString,
-        resolve: ({ sortable_id }) => sortable_id,
+        type: new GraphQLNonNull(GraphQLString),
       },
+      // sortableID: {
```
Comment (mzikherman) [inline src/schema/v2/aggregations/aggregation_count.ts]: 👍

**Comment by alloy** [comment]
lol, too late. Well, I think it should be fine still

**Comment by mzikherman** [comment]
Ah, whoops! Sorry, simultaneous action. Unfortunately we're not a single threaded simulation, at least. We've proven that.

Should this be reverted? This might break all betas filtering views.

**Comment by alloy** [comment]
I don’t actually expect anybody to be using those atm, so it’s fine imo

**Pull request merged**